src/js/wizards/Sell/6_Margin/components/MarginSelectorForm.jsx

```diff
@@ -1,53 +1,69 @@
 /* global web3 */
 import React, {Component, Fragment} from 'react';
 import PropTypes from 'prop-types';
-import {FormGroup, InputGroup, InputGroupAddon, ButtonGroup, InputGroupText} from 'reactstrap';
+import {FormGroup, InputGroup, InputGroupAddon, InputGroupText, Col, Row} from 'reactstrap';
 import Input from 'react-validation/build/input';
 import {withNamespaces} from 'react-i18next';
 import Form from 'react-validation/build/form';
-import CheckButton from '../../../../ui/CheckButton';
 import {isNumber, required, lowerThan} from '../../../../validators';
+import Slider from 'rc-slider/lib/Slider';
+import 'rc-slider/assets/index.css';
+import './MarginSelectorForm.scss';
 
 const ABOVE = 0;
 const BELOW = 1;
 
 class MarginSelectorForm extends Component {
+  onMarginChange = (value) => {
+    if (value < 0) {
+      if (this.props.marketType === ABOVE) {
+        this.props.marketTypeChange(BELOW);
```

Comment:
Maybe this could be refactored to not having to use a market type but read if the value is greater or minor than 0 to know the type of market.
what do you think?

Reply:
Yeah, that's pretty much what I did in the UI part of it.
Basically, the Container and actions are still using the market type, but the presentation layer works with -100 to 100 and converts to market type.
I agree that we could remove the market type, but I didn't want to have to modify the contract just now.
I'll add a task for it though.